src/main/java/io/github/darealturtywurty/ancientology/common/items/MjolnirItem.java

```diff
@@ -0,0 +1,27 @@
+package io.github.darealturtywurty.ancientology.common.items;
+
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.TranslatableComponent;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import net.minecraft.world.level.Level;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class MjolnirItem extends Item {
+    public MjolnirItem(Properties pProperties) {
+        super(pProperties);
+    }
+    @Override
+    public void appendHoverText(ItemStack pstack, @Nullable Level plevel,
+                                List<Component> pTooltipComponents, TooltipFlag pIsAdvanced){
+        if(Screen.hasShiftDown()){
+            pTooltipComponents.add(new TranslatableComponent("tooltip.ancientology.mjolnir"));
```

Comment:
This should really be a constant. Creating a new component every time this method is called, is very expensive.